aptos-move/framework/aptos-token-objects/sources/aptos_token.move

```diff
@@ -199,7 +199,7 @@ module aptos_token_objects::aptos_token {
         property_types: vector<String>,
         property_values: vector<vector<u8>>,
     ): ConstructorRef acquires AptosCollection {
-        let constructor_ref = token::create_from_account(
+        let constructor_ref = token::create_token(
```

Comment:
This needs to be behind the same feature flag as create_guid otherwise this would always fail before the flag is enabled

Reply:
If auids are enabled, I am now calling `create` method. If auids are disabled, I am calling `create_from_account` method. Is that okay?